src/commandDetails/coin/update.ts

```diff
@@ -23,7 +23,7 @@ const coinUpdateExecuteCommand: SapphireMessageExecuteType = async (client, mess
 
   // Second mandatory argument is amount
   const amount = args['amount'];
-  if (!amount) {
+  if (typeof amount != 'number') {
```

Comment:
`!==` should be used instead of `!=`.

Reply:
well at that line a change would only affect readability.